backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -331,7 +331,7 @@ and DType =
     | TError -> "Error"
     | THttpResponse _ -> "Response"
     | TDB _ -> "Datastore"
-    | TDate -> "Date"
+    | TDateTime -> "Date"
```

Comment:
Is this supposed to be changed to  `TDateTime -> "DateTime"`?

Reply:
yes

Reply:
Fixed